src/core/control/settings/SettingsEnums.h

```diff
@@ -77,6 +77,13 @@ enum IconTheme {
     ICON_THEME_LUCIDE = 1,
 };
 
+enum SidebarNumberingStyle {
+    SIDEBAR_NUM_STYLE_NONE = 0,
+    SIDEBAR_NUM_STYLE_1 = 1,
+    SIDEBAR_NUM_STYLE_2 = 2,
+    SIDEBAR_NUM_STYLE_3 = 3
```

Comment:
Could you either give more explicit names, or add a comment describing the styles here (or both)?

Reply:
Added some comments describing the different styles. See: [[4c6d713](https://github.com/xournalpp/xournalpp/pull/4693/commits/4c6d71340c245b0c841c3a02d9b1b5dc68529490)]